.slicemachine/libraries-state.json

```diff
@@ -103,7 +103,7 @@
         },
         "screenshotPaths": {
           "default": {
-            "path": "/Users/declan.ramsay/Documents/dev/quidditch/chaser/.slicemachine/assets/slices/LeagueTables/default/preview.png",
+            "path": "C:\\Users\\Michal\\Desktop\\Projects\\QUK\\chaser\\.slicemachine\\assets\\slices\\LeagueTables\\default\\preview.png",
```

Comment:
This looks like a known issue where this will change every time one of us makes a change to Slice Machine, so can you please add this file to .gitignore as that is the recommended fix for now

https://community.prismic.io/t/absolute-screenshotpaths-generate-in-slicemachine-libraries-states-json/8899/3

Reply:
Done :)